### PR TITLE
[Task] Add PHP CS Fixer

### LINKED ISSUE #6: [Github Action] Setup php-cs fixer

### PR DESCRIPTION
**Pull request by mattamon**
## Changes in this pull request  
Resolves #6 

## Additional info  
Setting up php cs fixer

### CODE REVIEW

File: .php-cs-fixer.dist.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    ->in([
+        __DIR__ . '/src',
+        __DIR__ . '/tests',
+        __DIR__ . '/config'
+    ])
+
+    ->exclude([
+        __DIR__ . '/tests/_output',
+        __DIR__ . '/tests/Support/_generated',
+    ])
```
Comment (martineiber) [inline .php-cs-fixer.dist.php:13]: Can be removed, since we have no tests.
```suggestion
```

File: .php-cs-fixer.dist.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()
+    ->in([
+        __DIR__ . '/src',
+        __DIR__ . '/tests',
```
Comment (martineiber) [inline .php-cs-fixer.dist.php:6]: Can also be removed.

```suggestion
```

**Review by martineiber — CHANGES_REQUESTED** [review]
We should add this line `/.php-cs-fixer.cache` to the `.gitignore`
Otherwise when running this locally, this file will be generated and can be committed.

File: .php-cs-fixer.dist.php (hunk 1 above)
Comment (mattamon) [reply .php-cs-fixer.dist.php:13]: But there will be tests, so I would keep them for now.

Comment (martineiber) [reply .php-cs-fixer.dist.php:13]: All right.. good to know.

**Review by martineiber — APPROVED** [review]

**Pull request merged**